### PR TITLE
fix shields and rewards not being translated to Norwegian

### PR DESCRIPTION
**Pull request by cezaraugusto**
fix https://github.com/brave/brave-browser/issues/3425

### Test Plan

1. Change your OS locale to Norwegian
2. Ensure both Shields and Rewards extensions are translated

### CODE REVIEW

File: BUILD.gn [hunk 1]
```diff
@@ -226,7 +226,6 @@ if (!is_mac && !is_android && !is_ios) {
           deps = [ "//brave/components/brave_rewards/resources/extension:locales_$locale" ]
 
           locale = string_replace(locale, "-", "_")
-          locale = string_replace(locale, "nb", "no")
```
Comment (bsclifton) [inline BUILD.gn]: why was this here in the first place?
cc: @simonhong

Comment (simonhong) [reply BUILD.gn]: honestly I can't remember why I did like that - Maybe I also did copy & paste from other places..

Comment (cezaraugusto) [reply BUILD.gn]: @bbondy thoughts on this? https://github.com/brave/brave-core/commit/cdbfca0d224f005bc16f974a7368d84d10dfca58#diff-71295ce804b64120ddc313f20e26c8adR135

Comment (bbondy) [reply BUILD.gn]: It used to be that way in Chromium as well, my guess is they changed it on one of the rebases and we didn't update our code accordingly.

**Comment by mkarolin** [comment]
Unfortunately, I think this breaks Transifex translation. The files in `nb` locale return untranslated (in English). I am assuming Transifex doesn't know what to do with this locale.
